addons/im_livechat/views/digest_views.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="digest_digest_view_form" model="ir.ui.view">
+        <field name="name">digest.digest.view.form.inherit.im.livechat</field>
```

Comment:
Guidlines : 
xml_id :  <module_name>.<extented_module_name>_<view_name>_inherit
name = xmlid with '.' instead of '_'
In this case : 
xml_id : im_livechat.digest_digest_view_form_inherit
name : im.livechat.digest.digest.view.form.inherit

Reply:
@dbeguin Done the changes. thanks